erasure-readfile_test.go

```diff
@@ -359,3 +362,65 @@ func TestErasureReadFileOffsetLength(t *testing.T) {
 		}
 	}
 }
+
+// Test erasureReadFile with random offset and lengths.
+// This test is t.Skip()ed as it a long time to run, hence should be run
+// explicitly after commenting out t.Skip()
+func TestErasureReadFileRandomOffsetLength(t *testing.T) {
+	// Comment the following line to run this test.
+	t.SkipNow()
+	// Initialize environment needed for the test.
+	dataBlocks := 7
```

Comment:
Correct, i have this on mind, it will be an exhaustive test coverage, will do this as a separate commit.